platform/source/mbed_os_timer.h

```diff
@@ -29,32 +30,65 @@ namespace mbed {
 namespace internal {
 
 #if MBED_CONF_RTOS_PRESENT
-#define OS_TICK_US (1000000 / OS_TICK_FREQ)
+typedef SysTimer<std::ratio<1, OS_TICK_FREQ>> OsTimer;
```

Comment:
Are you following some kind of standard for using vs typedef?

Reply:
No. Original code used `typedef`, and I didn't change it.

I'm quite happy changing to `using` - my only reservation is that Eclipse's outline view doesn't recognise it. (https://bugs.eclipse.org/bugs/show_bug.cgi?id=509120)